openedx/core/djangoapps/theming/storage.py

```diff
@@ -192,51 +192,56 @@ def converter(matchobj):
             This requires figuring out which files the matched URL resolves
             to and calling the url() method of the storage.
             """
-            matched, url = matchobj.groups()
+            matches = matchobj.groupdict()
```

Comment:
This code is from [django32](https://github.com/django/django/blob/3.2.22/django/contrib/staticfiles/storage.py#L163). They have made some changes in[ `django4.2`](https://github.com/django/django/blob/4.2.6/django/contrib/staticfiles/storage.py#L219). So updated this code also. Otherwise it gives errors with collectstatic command.

This is related [PR](https://github.com/django/django/pull/13843/files)